nipoppy/data/examples/sample_bids_dataset_description.json

```
@@ -0,0 +1,5 @@
+{
+    "Name": "CHANGEME",
+    "BIDSVersion": "1.10.0",
```

```suggestion
    "BIDSVersion": "1.10.1",
```

it was released now!  a "bugfix" which introduced many great features ;) https://bids-specification.readthedocs.io/en/stable/CHANGES.html#v1.10.1